package.json

```diff
@@ -20,7 +21,8 @@
     "preversion": "echo BE SURE TO UPDATE THE VERSION NUMBER IN SW.JS | chalk red bold",
     "version:patch": "npm --silent version patch",
     "version:minor": "npm --silent version minor",
-    "version:major": "npm --silent version major"
+    "version:major": "npm --silent version major",
+    "resize-images": "mogrify -resize 90x90 images/*.png; imagemin images/*.png --out-dir images"
```

Comment:
90 x 90 is probably too small... on my large monitor, they display at about 61 x 61 @2x, so 122 x 122.

You might consider using `srcset` and `sizes` to output small and large versions? 

More info in @grigs' endlessly helpful article series: https://cloudfour.com/thinks/responsive-images-101-definitions/

Reply:
I will look into that

Reply:
@tylersticka I have added a script to generate 1x and 2x versions (at 61x61 and 122x122), and now it is using `srcset`